portality/view/api_v1.py

```diff
@@ -40,61 +40,322 @@ def list_operations():
 def docs():
     return render_template('doaj/api_docs.html')
 
-@blueprint.route('/search/<search_type>/<path:search_query>')
-def search(search_type, search_query):
+@blueprint.route('/search/journals/<path:search_query>')
+def search_journals(search_query):
     """
-    Search journals and articles
+    Search for journals
     ---
     tags:
       - search
-    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In a augue nulla. In vehicula est eget erat iaculis, et lacinia enim faucibus. Aliquam diam dolor, euismod ultrices viverra vitae, congue sit amet massa. Duis vestibulum, quam ut ultrices sagittis, nunc diam cursus odio, sed efficitur eros erat a sapien."
     parameters:
       -
-        name: "search_type"
-        description: "lalalalallalala"
-        in: "path"
+        name: search_query
+        description: What you are searching for, e.g. computers
+        in: path
         required: true
-        type: "string"
+        type: string
       -
-        name: "search_query"
-        description: "this is definitely a very useful place to put some description of each parameter"
-        in: "path"
+        name: page
+        description: Which page of the results you wish to see. This field is optional.
```

Comment:
Is it necessary to say this field is optional? There is a required:false below it. If it is necessary (e.g. for visibility) then the pageSize one below, starting at line 64, does *not* have the sentence, but also has required:false declared.